atlp-rwanda/winners-c8-bn-fe
# #182210004 login front-end feature

**What does this PR do?**

Describe what PR do.

**Description of Task to be completed?**

1. Add a login form
2. Add validations
3. Handle server responses & behavior (states: wait, success, error)

**How should this be manually tested?**
1. clone this repository,  
2. add `PORT=5000` to .env file 
3. run `npm install && npm run dev`
4. access the app at `localhost:5000` and to to `login` tab to play with the form.
 for e.g: 
 - try using invalid email (without '@' sign for e.g) and see validations
 - try using `spaziltonx@soccerfit.com`  and `password: Simon@2022` and check the `auth-token` variable in the browser storage to see your token
 - etc .

**Any background context you want to provide?**
 N/A
**What are the relevant pivotal tracker stories?**
#182210004

**Screenshots (if appropriate)**
![image](https://user-images.githubusercontent.com/83488648/182045001-5235909e-7635-4267-8af1-5b7468a4cddb.png)


**Questions:**
N/A
